plugins/TwoFactorAuth/templates/userSettings.twig

```diff
@@ -0,0 +1,41 @@
+<div piwik-content-block
+     content-title="{{ 'TwoFactorAuth_TwoFactorAuthentication'|translate }} ({{ 'TwoFactorAuth_TwoFAShort'|translate }})" class="userSettings2FA">
+
+    <p>
+        {{ 'TwoFactorAuth_TwoFactorAuthenticationIntro'|translate('<a href="https://matomo.org/faq/general/faq_27245" rel="noreferrer noopener">', '</a>')|raw }}
+    </p>
+
+    {% if isEnabled %}
+        <p><strong class="twoFaStatusEnabled">{{ 'TwoFactorAuth_TwoFactorAuthenticationIsEnabled'|translate }}</strong></p>
+
+        <p>
+        {% if isForced %}
+            {{ 'TwoFactorAuth_TwoFactorAuthenticationRequired'|translate }}
+            <br />
+            <br />
+            <a class="btn btn-link enable2FaLink" href="{{ linkTo({'module': 'TwoFactorAuth', 'action': 'setupTwoFactorAuth'}) }}">{{ 'TwoFactorAuth_ConfigureDifferentDevice'|translate }}</a>
+        {% else %}
+            <a class="btn btn-link enable2FaLink" href="{{ linkTo({'module': 'TwoFactorAuth', 'action': 'setupTwoFactorAuth'}) }}">{{ 'TwoFactorAuth_ConfigureDifferentDevice'|translate }}</a>
+            <a href="{{ linkTo({'module': 'TwoFactorAuth', 'action': 'disableTwoFactorAuth', 'disableNonce': disableNonce}) }}" style="display:none;" id="disable2fa">disable2fa</a>
+            <input type="button"
+                   class="btn btn-link disable2FaLink"
+                   onclick="piwikHelper.modalConfirm('#confirmDisable2FA', {yes: function () { broadcast.propagateNewPage('module=TwoFactorAuth&action=disableTwoFactorAuth&disableNonce={{ disableNonce|e('url') }}'); }})"
```

Comment:
Should code this be in a JS file instead of the onclick attribute?

Reply:
I'll move the code within `onclick` into a JS file but will likely leave the onclick itself re simplicity...

Reply:
👍